src/Translator.php

```diff
@@ -298,6 +298,25 @@ private function mergeYaml(): void
                 $contentYaml = $this->arrayMergeRecursive($contentYaml, $parsedYaml);
                 $this->recursiveKeySort($contentYaml);
             }
+            // Remove any keys where the value is the same as the source english value
+            $enPath = preg_replace('#/[a-zA-Z_]+\.yml#', '/en.yml', $path);
+            if ($enPath !== $path) {
+                $enYaml = Yaml::parse(file_get_contents($enPath));
```

Comment:
Assign this variable outside the loop - we don't need to do this for every value in `$this->originalYaml`.